packages/cli/src/components/hooks/usePreviewHotkeys.tsx

```diff
@@ -69,13 +76,5 @@ export default function usePreviewHotkeys({ setViewMode }: Options) {
     [handleKey]
   );
 
-  const textareaRef = useCallback(
```

Comment:
didn't realize but we don't need this and it sends double keys... I think this changed when we made it non-editable